.eslintrc.json

```diff
@@ -0,0 +1,71 @@
+{
+	// Provide predefined global variables
```

Comment:
I didn't know JSON with comments was a thing! Is it an ESLint thing? A Node thing? What's the support like?

Reply:
Unfortunately (in our case, luckily), it's an ESLint thing.

Reply:
Does it support ESM? That might be preferable then, as it's a standard thing (and is basically the same code, just with an `export default` in the beginning.